system_tests/timeboost_test.go

```diff
@@ -35,11 +39,50 @@ import (
 	"github.com/offchainlabs/nitro/util/containers"
 	"github.com/offchainlabs/nitro/util/redisutil"
 	"github.com/offchainlabs/nitro/util/stopwaiter"
+	"github.com/offchainlabs/nitro/util/testhelpers"
 	"github.com/stretchr/testify/require"
 )
 
-func TestSequencerFeed_ExpressLaneAuction_ExpressLaneTxsHaveAdvantage(t *testing.T) {
+func TestTimeboostedInDifferentScenarios(t *testing.T) {
```

Comment:
This is more of a unit test than a system test and probably belongs in `nitro/broadcaster/message`